### PR TITLE
Add Pablo Iranzo Gómez repo for yaspeller

### PR DESCRIPTION
**Pull request by iranzo**
Add my repository for yaspeller hook

### CODE REVIEW

**Comment by asottile** [comment]
- it would be better to add the configuration to `yaspeller` directly
- worst case, it would be better to have a [mirror repository](https://github.com/pre-commit/pre-commit-mirror-maker) instead of a floating version

**Comment by iranzo** [comment]
> * it would be better to add the configuration to `yaspeller` directly

The configuration can be done via .yaspeller.json directly, in fact the repository itself uses it with pre-commit hook

> * worst case, it would be better to have a [mirror repository](https://github.com/pre-commit/pre-commit-mirror-maker) instead of a floating version

I've been reading that repository but from the README it's not clear to me what is for.

It states `Scripts for creating mirror repositories that do not have .pre-commit-hooks.yaml` but then in the folder listing it shows the `.pre-commit-hooks.yaml`

We can work on that via an issue on the repo for the yaspeller hook, is that ok?

**Comment by asottile** [comment]
no no, I mean the *pre-commit* configuration -- instead of your repo it would be best to contribute a `.pre-commit-hooks.yaml` to yaspeller's repository directly

**Comment by iranzo** [comment]
Ah, got it, let me PR them

**Comment by iranzo** [comment]
@asottile something like this? https://github.com/hcodes/yaspeller/pull/136

**Comment by iranzo** [comment]
@asottile PR has been updated to point to yaspeller repository

File: all-repos.yaml [hunk 1]
```diff
@@ -122,3 +122,4 @@
 - https://github.com/talos-systems/conform
 - https://github.com/twu/skjold
 - https://github.com/woile/commitizen
+- https://github.com/hcodes/yaspeller.git
```
Comment (asottile) [inline all-repos.yaml]: none of the others end with .git -- why did you do it for this one?

Comment (iranzo) [reply all-repos.yaml]: It was giving me a failure when I tested it , I can remove it and leave like the others

Comment (asottile) [reply all-repos.yaml]: looks good but have to resolve the conflicts

Comment (iranzo) [reply all-repos.yaml]: done!

**Comment by mxr** [comment]
This was added in #344

**Pull request closed without merging**